packages/loading-skeleton/docs/LoadingSkeleton.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react"
+import { Story } from "@storybook/react"
+import { withDesign } from "storybook-addon-designs"
+import { Paragraph } from "@kaizen/typography"
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+import { LoadingParagraph } from "../src/LoadingSkeleton"
+import { figmaEmbed } from "../../../storybook/helpers"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.loadingSkeleton}`,
```

Comment:
Based on the specs of your task, since the Loading components will have different props, you'll probably want different stories files so that the ArgsTable can correctly display the props you have access too, so I suggest renaming this file to `LoadingParagraph.stories.tsx`.

We can look at trying to combining and optimising stories in Storybook later :)
I think if you keep the `title` the same, they should combine in the tree at the very least.

Reply:
Updated to LoadingParagraph and I agree once I've built them out I intend on finding a nice way to put them together.